### PR TITLE
🔖(patch) bump release to 1.2.1

### PR DESCRIPTION
**Pull request by quitterie-lcs**
## [1.2.1] - 2024-03-18

 Added:

- Enable email lookup for Keycloak-OAuth

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,7 +8,9 @@ Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
 
 ## [Unreleased]
 
-## Added
+## [1.2.1] - 2024-03-18
```
Comment (lunika) [inline CHANGELOG.md:11]: You also have to update the end of this file to create the compare link with this new release

**Review by lunika — COMMENTED** [review]
I also see there a `VERSION` file you have to update.
If you take a look to this commit you will see there are also other files to update: https://github.com/openfun/potsie/commit/d2d3c11bfd9a6a58081fb4246072ce9c84270c63

File: CHANGELOG.md (hunk 1 above)
Comment (quitterie-lcs) [reply CHANGELOG.md:11]: Wooopsie, I forgot it

**Comment by quitterie-lcs** [comment]
Wooopsie, I forgot everything, I wasn't focused

**Review by lunika — APPROVED** [review]

**Pull request merged**